app/models/entities/Enums.scala

```diff
@@ -0,0 +1,7 @@
+package models.entities
+
```

Comment:
This is fine how it is. Another way you'll often see this done in Scala is using a sealed trait. It would look something like:
```
sealed trait Comparison
case object LessThan extends Comparison
case object GreaterThan extends Comparison
```
The `sealed` keyword tells the compiler that all classes that extend the trait are defined in that file. It allows you to match exhaustively so you can have something like:
```
def fun(c: Comparator): Boolean = c match {
  case GreaterThan => ???
  case LessThan => ???
}
```
Without having to include a `case _ => ???` to catch any remaining cases, since the compiler can infer that all valid cases are represented.

As I said, there is no need to change this, I just wanted you to be aware of how you might see the Enum idea implemented elsewhere in the project.